core/src/main/java/io/questdb/PropServerConfiguration.java

```diff
@@ -430,6 +433,9 @@ public PropServerConfiguration(
         this.snapshotInstanceId = getString(properties, env, PropertyKey.CAIRO_SNAPSHOT_INSTANCE_ID, "");
         this.snapshotRecoveryEnabled = getBoolean(properties, env, PropertyKey.CAIRO_SNAPSHOT_RECOVERY_ENABLED, true);
         this.simulateCrashEnabled = getBoolean(properties, env, PropertyKey.CAIRO_SIMULATE_CRASH_ENABLED, false);
+        this.checkOsProcessLimits = getBoolean(properties, env, PropertyKey.CAIRO_CHECK_OS_PROCESS_LIMITS, true);
+        this.checkOsProcessLimitFiles = getLong(properties, env, PropertyKey.CAIRO_CHECK_OS_PROCESS_FILES_LIMIT, 100_000);
```

Comment:
naming: `OS` is redundant

Reply:
done in https://github.com/questdb/questdb/pull/2289/commits/5bfdd906251b7ede8d75aa234a5ccfff1892dcec